zanata-war/src/main/java/org/zanata/webtrans/server/rpc/AbstractActionHandler.java

```diff
@@ -15,7 +15,9 @@
         implements ActionHandler<A, R> {
 
     protected AbstractActionHandler() {
-        assert getActionType() != null;
+        // TODO This is a sanity check only. It has been disabled to allow for
+        // CDIUnit tests
+        //assert getActionType() != null;
```

Comment:
It was, but I actually found a way to re-enable it.

Turns out I just needed to decorate the `@ActionHandlerFor` annotation with `@Inherited` in order for proxies to get the annotation as well.